event.c

```diff
@@ -249,3 +249,44 @@ ULONGLONG GetTestEventHandle() {
 	EvtClose(result);
 	return (ULONGLONG)record;
 }
+
+ULONGLONG EnableChannel(char* ChannelName, int BufferSizeMb) {
```

Comment:
It would be more convinient to have both `Enable` and `Disable` methods in Go code, so we need `BOOL enable` param.